README.md

```diff
@@ -69,6 +69,12 @@ app.prepare().then(() => {
 })
 
 ```
+Optionally you can pass a custom handler, for example:
+```javascript
+const handler = routes.getRequestHandler(app, ({req, res, route, query}) => {
+  renderAndCache(req, res, route.page, query)
```

Comment:
I recommend removing `renderAndCache` and instead show how to properly handle the route manually. (```app.render(...)```)